bundle/Cache/SwitchableHttpCachePurger.php

```diff
@@ -42,12 +42,12 @@ public function purgeAll()
         $this->gatewayCachePurger->purgeAll();
     }
 
-    public function purgeForContent($contentId)
+    public function purgeForContent($contentId, $locationIds = array())
```

Comment:
yes, _but_ adding optional argument won't break the contract. In retrospective it was the kernel that broke the contract here.